{"d89e5d3dcae2f873575c0ac61689d19c":{"statistic":{"failed":1,"broken":7,"skipped":0,"passed":0,"unknown":0,"total":8},"items":[{"uid":"7025c61bbbc7da36","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/32//#testresult/7025c61bbbc7da36","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x564daaa48b13 <unknown>\n#1 0x564daa84f688 <unknown>\n#2 0x564daa873f1f <unknown>\n#3 0x564daa86f5aa <unknown>\n#4 0x564daa8aa64a <unknown>\n#5 0x564daa8a47a3 <unknown>\n#6 0x564daa87a0ea <unknown>\n#7 0x564daa87b225 <unknown>\n#8 0x564daaa902dd <unknown>\n#9 0x564daaa942c7 <unknown>\n#10 0x564daaa7a22e <unknown>\n#11 0x564daaa950a8 <unknown>\n#12 0x564daaa6ebc0 <unknown>\n#13 0x564daaab16c8 <unknown>\n#14 0x564daaab1848 <unknown>\n#15 0x564daaacbc0d <unknown>\n#16 0x7fd19c9af609 <unknown>","time":{"start":1657905362692,"stop":1657905364048,"duration":1356}},{"uid":"abf0febb5e55d6a9","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/31//#testresult/abf0febb5e55d6a9","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657904731206,"stop":1657904734492,"duration":3286}},{"uid":"3a2dc4ea95bdb25d","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/30//#testresult/3a2dc4ea95bdb25d","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x5625dc63cb13 <unknown>\n#1 0x5625dc443688 <unknown>\n#2 0x5625dc467f1f <unknown>\n#3 0x5625dc4635aa <unknown>\n#4 0x5625dc49e64a <unknown>\n#5 0x5625dc4987a3 <unknown>\n#6 0x5625dc46e0ea <unknown>\n#7 0x5625dc46f225 <unknown>\n#8 0x5625dc6842dd <unknown>\n#9 0x5625dc6882c7 <unknown>\n#10 0x5625dc66e22e <unknown>\n#11 0x5625dc6890a8 <unknown>\n#12 0x5625dc662bc0 <unknown>\n#13 0x5625dc6a56c8 <unknown>\n#14 0x5625dc6a5848 <unknown>\n#15 0x5625dc6bfc0d <unknown>\n#16 0x7fec7524e609 <unknown>","time":{"start":1657904310521,"stop":1657904312095,"duration":1574}},{"uid":"ad1ffcd428820079","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/29//#testresult/ad1ffcd428820079","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55ccdb1a4b13 <unknown>\n#1 0x55ccdafab688 <unknown>\n#2 0x55ccdafcff1f <unknown>\n#3 0x55ccdafcb5aa <unknown>\n#4 0x55ccdb00664a <unknown>\n#5 0x55ccdb0007a3 <unknown>\n#6 0x55ccdafd60ea <unknown>\n#7 0x55ccdafd7225 <unknown>\n#8 0x55ccdb1ec2dd <unknown>\n#9 0x55ccdb1f02c7 <unknown>\n#10 0x55ccdb1d622e <unknown>\n#11 0x55ccdb1f10a8 <unknown>\n#12 0x55ccdb1cabc0 <unknown>\n#13 0x55ccdb20d6c8 <unknown>\n#14 0x55ccdb20d848 <unknown>\n#15 0x55ccdb227c0d <unknown>\n#16 0x7fe86e688609 <unknown>","time":{"start":1657903186502,"stop":1657903188024,"duration":1522}},{"uid":"179229099d4919e9","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/28//#testresult/179229099d4919e9","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x56439b02eb13 <unknown>\n#1 0x56439ae35688 <unknown>\n#2 0x56439ae59f1f <unknown>\n#3 0x56439ae555aa <unknown>\n#4 0x56439ae9064a <unknown>\n#5 0x56439ae8a7a3 <unknown>\n#6 0x56439ae600ea <unknown>\n#7 0x56439ae61225 <unknown>\n#8 0x56439b0762dd <unknown>\n#9 0x56439b07a2c7 <unknown>\n#10 0x56439b06022e <unknown>\n#11 0x56439b07b0a8 <unknown>\n#12 0x56439b054bc0 <unknown>\n#13 0x56439b0976c8 <unknown>\n#14 0x56439b097848 <unknown>\n#15 0x56439b0b1c0d <unknown>\n#16 0x7f7f3da3b609 <unknown>","time":{"start":1657896733644,"stop":1657896734633,"duration":989}},{"uid":"26a72630e0f8a979","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/27//#testresult/26a72630e0f8a979","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657839031743,"stop":1657839031744,"duration":1}},{"uid":"de7b908efc2383fb","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/26//#testresult/de7b908efc2383fb","status":"broken","statusDetails":"NameError: name 'driver' is not defined","time":{"start":1657838860891,"stop":1657838860891,"duration":0}},{"uid":"ae56afc0d717c14","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/25//#testresult/ae56afc0d717c14","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657838605942,"stop":1657838605943,"duration":1}}]},"62b43e6912a64c8eaa3d9e86da5a604e":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"d572fe9fbb7673ad","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/24//#testresult/d572fe9fbb7673ad","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55866afc2b13 <unknown>\n#1 0x55866adc9688 <unknown>\n#2 0x55866adedf1f <unknown>\n#3 0x55866ade95aa <unknown>\n#4 0x55866ae2464a <unknown>\n#5 0x55866ae1e7a3 <unknown>\n#6 0x55866adf40ea <unknown>\n#7 0x55866adf5225 <unknown>\n#8 0x55866b00a2dd <unknown>\n#9 0x55866b00e2c7 <unknown>\n#10 0x55866aff422e <unknown>\n#11 0x55866b00f0a8 <unknown>\n#12 0x55866afe8bc0 <unknown>\n#13 0x55866b02b6c8 <unknown>\n#14 0x55866b02b848 <unknown>\n#15 0x55866b045c0d <unknown>\n#16 0x7f5b71edf609 <unknown>","time":{"start":1657831041449,"stop":1657831042960,"duration":1511}},{"uid":"40eb5480234ec27e","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/23//#testresult/40eb5480234ec27e","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55e1825d2b13 <unknown>\n#1 0x55e1823d9688 <unknown>\n#2 0x55e1823fdf1f <unknown>\n#3 0x55e1823f95aa <unknown>\n#4 0x55e18243464a <unknown>\n#5 0x55e18242e7a3 <unknown>\n#6 0x55e1824040ea <unknown>\n#7 0x55e182405225 <unknown>\n#8 0x55e18261a2dd <unknown>\n#9 0x55e18261e2c7 <unknown>\n#10 0x55e18260422e <unknown>\n#11 0x55e18261f0a8 <unknown>\n#12 0x55e1825f8bc0 <unknown>\n#13 0x55e18263b6c8 <unknown>\n#14 0x55e18263b848 <unknown>\n#15 0x55e182655c0d <unknown>\n#16 0x7fb79cd5c609 <unknown>","time":{"start":1657830368832,"stop":1657830372761,"duration":3929}}]},"4db9caae8ef7326232c323be69d7c921":{"statistic":{"failed":5,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":5},"items":[{"uid":"beec529e99322af5","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/24//#testresult/beec529e99322af5","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657831041419,"stop":1657831041419,"duration":0}},{"uid":"3cf05b2daf80b506","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/23//#testresult/3cf05b2daf80b506","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657830368801,"stop":1657830368802,"duration":1}},{"uid":"d9f492d8c5490526","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/22//#testresult/d9f492d8c5490526","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657742529712,"stop":1657742529712,"duration":0}},{"uid":"4b4a9418cbf6ae3b","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/20//#testresult/4b4a9418cbf6ae3b","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657741951648,"stop":1657741951648,"duration":0}},{"uid":"6c3cb921775d6672","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/19//#testresult/6c3cb921775d6672","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657730708932,"stop":1657730708933,"duration":1}}]},"2de973a726f6108f7c3b207b90ba2fc4":{"statistic":{"failed":0,"broken":7,"skipped":0,"passed":1,"unknown":0,"total":8},"items":[{"uid":"1a3f9e0f7bfd8273","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/32//#testresult/1a3f9e0f7bfd8273","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x561f4c384b13 <unknown>\n#1 0x561f4c18b688 <unknown>\n#2 0x561f4c1aff1f <unknown>\n#3 0x561f4c1ab5aa <unknown>\n#4 0x561f4c1e664a <unknown>\n#5 0x561f4c1e07a3 <unknown>\n#6 0x561f4c1b60ea <unknown>\n#7 0x561f4c1b7225 <unknown>\n#8 0x561f4c3cc2dd <unknown>\n#9 0x561f4c3d02c7 <unknown>\n#10 0x561f4c3b622e <unknown>\n#11 0x561f4c3d10a8 <unknown>\n#12 0x561f4c3aabc0 <unknown>\n#13 0x561f4c3ed6c8 <unknown>\n#14 0x561f4c3ed848 <unknown>\n#15 0x561f4c407c0d <unknown>\n#16 0x7fd062ac8609 <unknown>","time":{"start":1657905359491,"stop":1657905362602,"duration":3111}},{"uid":"ec5afd89a43d633","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/31//#testresult/ec5afd89a43d633","status":"passed","time":{"start":1657904722269,"stop":1657904731203,"duration":8934}},{"uid":"54c1bdd72f8fffd2","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/30//#testresult/54c1bdd72f8fffd2","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55c02ece5b13 <unknown>\n#1 0x55c02eaec688 <unknown>\n#2 0x55c02eb10f1f <unknown>\n#3 0x55c02eb0c5aa <unknown>\n#4 0x55c02eb4764a <unknown>\n#5 0x55c02eb417a3 <unknown>\n#6 0x55c02eb170ea <unknown>\n#7 0x55c02eb18225 <unknown>\n#8 0x55c02ed2d2dd <unknown>\n#9 0x55c02ed312c7 <unknown>\n#10 0x55c02ed1722e <unknown>\n#11 0x55c02ed320a8 <unknown>\n#12 0x55c02ed0bbc0 <unknown>\n#13 0x55c02ed4e6c8 <unknown>\n#14 0x55c02ed4e848 <unknown>\n#15 0x55c02ed68c0d <unknown>\n#16 0x7f0a90886609 <unknown>","time":{"start":1657904306984,"stop":1657904310433,"duration":3449}},{"uid":"dbe9e4bd31b5095c","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/29//#testresult/dbe9e4bd31b5095c","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55af1bf16b13 <unknown>\n#1 0x55af1bd1d688 <unknown>\n#2 0x55af1bd41f1f <unknown>\n#3 0x55af1bd3d5aa <unknown>\n#4 0x55af1bd7864a <unknown>\n#5 0x55af1bd727a3 <unknown>\n#6 0x55af1bd480ea <unknown>\n#7 0x55af1bd49225 <unknown>\n#8 0x55af1bf5e2dd <unknown>\n#9 0x55af1bf622c7 <unknown>\n#10 0x55af1bf4822e <unknown>\n#11 0x55af1bf630a8 <unknown>\n#12 0x55af1bf3cbc0 <unknown>\n#13 0x55af1bf7f6c8 <unknown>\n#14 0x55af1bf7f848 <unknown>\n#15 0x55af1bf99c0d <unknown>\n#16 0x7f158a993609 <unknown>","time":{"start":1657903183177,"stop":1657903186417,"duration":3240}},{"uid":"6b64e6a8b72ec266","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/28//#testresult/6b64e6a8b72ec266","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55b624426b13 <unknown>\n#1 0x55b62422d688 <unknown>\n#2 0x55b624251f1f <unknown>\n#3 0x55b62424d5aa <unknown>\n#4 0x55b62428864a <unknown>\n#5 0x55b6242827a3 <unknown>\n#6 0x55b6242580ea <unknown>\n#7 0x55b624259225 <unknown>\n#8 0x55b62446e2dd <unknown>\n#9 0x55b6244722c7 <unknown>\n#10 0x55b62445822e <unknown>\n#11 0x55b6244730a8 <unknown>\n#12 0x55b62444cbc0 <unknown>\n#13 0x55b62448f6c8 <unknown>\n#14 0x55b62448f848 <unknown>\n#15 0x55b6244a9c0d <unknown>\n#16 0x7fd9e9b70609 <unknown>","time":{"start":1657896732163,"stop":1657896733533,"duration":1370}},{"uid":"3fd112bad02d1f61","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/27//#testresult/3fd112bad02d1f61","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657839031623,"stop":1657839031624,"duration":1}},{"uid":"2463d8e9947a801c","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/26//#testresult/2463d8e9947a801c","status":"broken","statusDetails":"NameError: name 'driver' is not defined","time":{"start":1657838860866,"stop":1657838860867,"duration":1}},{"uid":"ae55eb0012aa2a06","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/25//#testresult/ae55eb0012aa2a06","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657838605845,"stop":1657838605846,"duration":1}}]}}